addons/switchHeadgear/XEH_preInit.sqf

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+ADDON = false;
+
+PREP_RECOMPILE_START;
+#include "XEH_PREP.hpp"
+PREP_RECOMPILE_END;
+
+ADDON = true;
+
+[
+    QGVAR(enableSwitch),
+    "CHECKBOX",
+    localize LSTRING(settingEnable_displayName),
+    localize "STR_grad_settingsCategory",
```

Comment:
In versionCheck hab ich ne eigene Kategorie für die Komponente aufgemacht:
https://github.com/gruppe-adler/gruppe_adler_mod/blob/9d352000c6441e958c3f5abebb29b35d1cf0630c/addons/versionCheck/XEH_preInit.sqf#L17-L24

Find ich glaub ich übersichtlicher, wenn da später noch mehr dazukommt, selbst wenn man wie hier nur ein einziges Setting hat.

Reply:
Wäre es dann nicht besser, einzelne Settings in einer allgemeinen Kategorie zu behalten und sobald man dann mehr hat, ne Eigene aufzumachen? Weil ansonsten hat man noch mehr Kategorien, neben ACE usw.

Reply:
Meinetwegen. versionCheck Kategorie über diesen PR angleichen?

Reply:
Meine Idee war eigentlich, dass man alle Komponenten mit <= 2 Settings in eine Hauptkategorie zusammenfasst und dem Rest eine eigene Kategorie gibt. So in etwa wie bei ACE Common